rasterio/windows.py

```diff
@@ -195,6 +213,33 @@ def from_bounds(left, bottom, right, top, transform,
         return crop(window, height, width)
 
 
+def int_reshape(window, pixel_precision=3):
+    """Converts floating point value Windows to integer value Windows.
+
+    Parameters
+    ----------
+    window : Window
```

Comment:
@sgillies potential consistency issue, most of your other functions below say `window: Window or window tuple`

Which itself begs a different question: for your docstrings, since window tuples are now deprecated, should they just list the preferred input format?